Client/StyleSuppressions.cs

```diff
@@ -0,0 +1,9 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
```

Comment:
We have internal consensus on not using these files. The same can be achieved with configurations in the `.editorconfig` file.
PS Comment applies to all such files in the PR.